### PR TITLE
Add a kubernetes example

### LINKED ISSUE #64: README: provide a kubernetes pod example
This will give people a fast onboard to trying it out and show them how pods can be used to protect a service.

### PR DESCRIPTION
**Pull request by Quentin-M**
Fixes #64

### CODE REVIEW

**Comment by ecordell** [comment]
Why not run the signer in another pod?

Then you could `curl -x http://<kube_ip>:<signer_port> http://<kube_ip>:<verifier_port>`. Not a big difference but maybe it would emphasize the service-to-service use case.

**Comment by Quentin-M** [comment]
@ecordell That was to simplify a bit but ok, can do.

**Comment by Quentin-M** [comment]
@ecordell Updated :)

File: examples/kubernetes-httpserver/tester-app.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - /etc/jwtproxy/signer.yaml
+    volumeMounts:
+    - mountPath: /etc/jwtproxy/
+      name: secret-tester-volume
+  - name: tester
+    image: alpine:edge
+    command:
+    - sh
+    - -c
+    - while :; do http_proxy=http://localhost:3128 wget -qO- -Y on http://nginx/; sleep 5; done
```
Comment (ecordell) [inline examples/kubernetes-httpserver/tester-app.yaml]: 👍

**Comment by ecordell** [comment]
LGTM

**Pull request merged**